docs/embedded/concepts/spe-copilot.md

```diff
@@ -0,0 +1,262 @@
+
+6. You can follow the instrucitons of the `README.md` file in the root of the project for further npm commands. Run `npm run start` in the root of the project to start your application with the SPE copilot functionality enabled.
+
+![VS Code terminal in root folder of SPE Typescript project cloned earlier and npm run start command typed in](../images/speco-runnpmrunstart.png)
+
+7. Sign in with a copilot license enabled user
+
+![SPE Typescript App running in Edge with sign in buttons](../images/speco-reacttypescripthomepage.png)
+
+8. Navigate to the `containers` page, create one if you do not have any yet
+
+![SPE Typescript App running in edge in /containers sub page with modal of user c reatign a container called ContosoCompanyContainer](../images/speco-createcontosocontainer2.png)
+
+After it's created you will see it here
+
+![SPE Typescript App running in edge with a created container from above ContosoCompanyContainer](../images/speco-createdcontainer.png)
+
+9. Click the container and upload your files. Once a container has been created and you have navigated inside it, your copilot chat experience will become enabled.   
+
+![SPE Typescript App running in edge inside a created container page of ContosoCompanyContainer](../images/speco-spechatenabled.png)
+
+### Examples
+
+### SPE Typescript React Application
+
+### Customer Succes Stories
+
+# API Documentation
+
+[LINK_TO_API_EXTRACTOR](https://api-extractor.com/pages/setup/generating_docs/)
+
+# Frequently Asked Questions
+
+## Consumptive Billing
+
+Currently you need a copilot license enabled for your user to use Sharepoint Embedded copilot. When consumptive billing is enabled you will no longer require a license however you will be required to use a Standard Container type.
+
+***Trial Container Types expire after 30 days, for this reason we recommend starting off with Standard Container types. Currently there is no upgrade path from Trial to Standard container types.***
+
+## Standard or Trail Container Type
+
+Once consumptive billing is enabled, we will be disabling using this feature with Trial Container types and it will only be enabled on Standard container types going forward. Please follow this [guide](../concepts/app-concepts/containertypes.md) to get startted on creating your Standard Container type.
+
+## Bring-Your-Own-Model
+
+## Language/Locale
+
+## Authentication and 3P Cookies
+
+# Support
+
+## Chat Control Feedback Dialog
+
+If you encounter any issues with the chat control, please use the thumbs up and down feedback buttons to report the problem. This method is preferred for sending feedback because it provides us with telemetry data that helps us diagnose and troubleshoot the issue more effectively.
+
+![alt text](../images/speco-feedbackthumbsdown.png)
+
+When you click the thumbs down button, a feedback dialog will appear. Please include any relevant information in this dialog.
+
+![alt text](../images/speco-feedbackmodal.png)
+
+## Contact Us

```

Comment:
Remove--do NOT add this publicly. All feedback to go through official channels and NOT direct email.